components/utils/src/templates.rs

```diff
@@ -20,7 +20,9 @@ pub fn render_template(name: &str, tera: &Tera, context: &Context, theme: Option
     }
 
     if name == "index.html" || name == "section.html" || name == "page.html" {
-        return Ok(String::new());
+        return tera
+            .render("index.html", &{})
+            .map_err(|e| e.into());
```

Comment:
In the end I think having one default file for each would be better as it allows us to give more descriptive message and we can remove those lines entirely. Right now it would say to modify `index.html` for sections and pages

Reply:
so, another file in templates for section and pages?

Reply:
Another one for each yes so a `section.html` and `page.html`. They can be mostly copy/pasted of the same with changing the template name and doc links